backend/README.md

```diff
@@ -5,6 +5,7 @@
 - `DATABASE_URL`: The URL of the database to connect to. Example: `postgres://user:password@localhost:5432/database`
 - `BACKEND_ADDRESS`: The address to bind the backend server to. Example: `127.0.0.1:8080`
 - `OPENAPI_JSON_FRONTEND_PATH`: The path to write openapi json in frontend, relative to backend/ directory. Example: `../frontend/openapi/api/openapi.json`
+- `RUST_LOG`: The log level for crates. [Docs](https://docs.rs/log/latest/log/enum.Level.html). Example: `hack4krak_backend=trace,actix-web=info`
```

Comment:
Link to https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html